### PR TITLE
Add Mar 2021 meeting agenda/minutes

### PR DESCRIPTION
**Pull request by aays**

### CODE REVIEW

File: meetings/Agenda_2021-03-24.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Meeting Agenda, Mar 24 2021
+
+1. **Debrief** (20 min)
+ * What did we feel went well about this past semester in terms of lessons?
+ * What could have been better about the lessons this past semester? 
+ * How did we feel about the change in schedule this semester?
+ * How did we feel about co-working? 
+
+2. **Summer/future plans** (20-25 min)
```
Comment (FrancesWong) [inline meetings/Agenda_2021-03-24.md:9]: Carpentries workshop in the summer, May?

File: meetings/Agenda_2021-03-24.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+1. **Debrief** (20 min)
+ * What did we feel went well about this past semester in terms of lessons?
+ * What could have been better about the lessons this past semester? 
+ * How did we feel about the change in schedule this semester?
+ * How did we feel about co-working? 
+
+2. **Summer/future plans** (20-25 min) 
+ * Potential new execs to bring on board
+ * How does each of us feel about sticking around for another year? 
+
```
Comment (HaidyGiratallah) [inline meetings/Agenda_2021-03-24.md:12]: 3. Ulife status. How do we get back on track?

File: meetings/Agenda_2021-03-24.md (hunk 1 above)
Comment (aays) [reply meetings/Agenda_2021-03-24.md:9]: GAH I knew I was forgetting something thank you

File: meetings/Agenda_2021-03-24.md (hunk 2 above)
Comment (aays) [reply meetings/Agenda_2021-03-24.md:12]: Will add, good call!

**Pull request merged**